azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/redux/state/CallingState.kt

```diff
@@ -24,6 +26,7 @@ internal data class CallingState(
     val joinCallIsRequested: Boolean = false,
     val isRecording: Boolean = false,
     val isTranscribing: Boolean = false,
+    val callStartLocalDateTime: OffsetDateTime? = null,
```

Comment:
rename; also, document that this is fixed for the duration of the call (in a reducer's handling of CallStart event)

Reply:
Done